### ISSUE TITLE
Extracting msi: error writing: no space...

### ISSUE THREAD

**Issue by kristof-linux** [linux-surface/surface-uefi-firmware#28]
Hello,
I'm trying to extract the msi-surface file using your script, but i get error writing to file: no space left on device.
Please can you help to solve this? Did i everything correct please - see attachment. Thx so much, Kristof

![image](https://github.com/linux-surface/surface-uefi-firmware/assets/58733661/4907a6ec-f864-4dd1-b69f-c8d7c1b52a6d)
![image](https://github.com/linux-surface/surface-uefi-firmware/assets/58733661/77b0dfc9-ec13-4dcb-9bec-d65cfd8a02ef)
![image](https://github.com/linux-surface/surface-uefi-firmware/assets/58733661/faa9ce00-c67e-4ee2-a604-6fb798709d8d)

**Comment by StollD** [comment]
Yeah this is an issue I ran into yesterday too. The reason for it is that MS made these MSI packages so big they dont fit into your memory anymore (they are extracted to `/tmp` which is a ramdisk).

I've pushed a fix that lets it extract the MSI in the current directory instead.

**Comment by kristof-linux** [comment]
Thx a lot, this worked!

Just a question for good understanding (i'm not so experienced in linux): this script will run automatically to check if there are more updates? Or how does it work exactly? Thx a lot for your help!

gr,

Kristof
